content/blog/2020-03-06-die-dynamik-von-G-kräften-betriebssystemen-und-neuen-teammitgliedern.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Die Dynamik von G-Kräften, Betriebssystemen und neuen Teammitgliedern
+slug: die-dynamik-von-G-kräften-betriebssystemen-und-neuen-teammitgliedern
+templateKey: blog-post
+image: ./img/renato-snowboarding.jpg
```

Comment:
Ich würde den Blog in eine separaten Ordner mit dem Bild tun. So ist es etwas aufgeräumter. Die ohne Ordner sind noc die alten aus dem Wordpress exportiert.